src/Widgets/ToolBars/ImageToolBar.vala

```diff
@@ -30,13 +30,13 @@ namespace Writer.Widgets {
             get_style_context ().add_class ("writer-toolbar");
 
             var wrap_combobox = new Gtk.ComboBoxText ();
-            wrap_combobox.append ("In line of text", ("In line of text"));
-            wrap_combobox.append ("Float above text", ("Float above text"));
+            wrap_combobox.append (_("In line of text"), ("In line of text"));
```

Comment:
```suggestion
            wrap_combobox.append ("In line of text", _("In line of text"));
```

The first argument of `append ()` method of `Gtk.ComboBoxTest` is an id string to distinguish which text currently selected, so it should not be translatable. Instead, the second argument is shown in UI and should be translatable.
See https://valadoc.org/gtk+-3.0/Gtk.ComboBoxText.append.html for more details.

Reply:
I wasn't sure about which one was the string that should be translated thanks for your explanation.